bin/dry-run.js

```diff
@@ -43,7 +43,7 @@ const fetchMeasurements = async (cid) => {
     if (err.code !== 'ENOENT') console.warn('Cannot read cached measurements:', err)
   }
 
-  const measurements = await fetchMeasurementsViaGateway(cid)
+  const measurements = await fetchMeasurementsTrusted(cid)
```

Comment:
Can we use `fetchMeasurementsTrustless` here too?

I added `fetchMeasurementsViaGateway` to allow people (including myself) to run this code without WEB3_STORAGE_API_TOKEN, and I was too lazy to implement streaming CAR validation. Now that we have the validation, I think there is no reason for not using it.

Reply:
Awesome, yes let's do that!